app/models/concerns/mergeable.rb

```diff
@@ -82,9 +82,12 @@ def update_existing_record(existing_record, attributes)
       existing_record
     end
 
-    def return_old_record(existing_record)
+    def return_old_record(existing_record, new_record)
       logger.debug "#{self} with id #{existing_record.id} is old, keeping existing."
       increment_metric(:old)
+      if existing_record.device_updated_at == new_record.device_updated_at
+        increment_metric(:same_device_updated_at)
+      end
       existing_record.touch
```

Comment:
Were we considering doing something about this touch? I'm guessing you just want to log for now to see if this is the cause before we revert the touch.

Reply:
Yup, just tracking for now to see what things look like tomorrow.